src/Resources/views/storefront/layout/meta.html.twig

```diff
@@ -94,6 +96,15 @@
                 window.EnderecoIntegrator.config.ux.checkExisting = false;
                 window.EnderecoIntegrator.config.ux.resumeSubmit = !!('{{ page.endereco_config.enderecoContinueSubmit }}');
                 window.EnderecoIntegrator.config.ux.useStandardCss = !!('{{ page.endereco_config.enderecoUseDefaultCss }}');
+
+                // Check if the setting specifies inline loading or file loading.
+                // TODO: @michal, get the setting from plugin settings, please.
+                if (true) {
```

Comment:
@michal127 please, add a select to advanced setting with two values:
1. Load CSS from file
2. Load CSS inline
(3. Dont load CSS at all)

Default being "Load CSS from file".

We could, if it is doable easily, remove the enderecoUseDefaultCss setting and replace it with the loading type setting.